### DISCUSSION TITLE
Sujet / Thema 8 –– Die Marke «NGDI» / La marque «INDG»

### DISCUSSION THREAD

**Discussion by rastrau** [GeoUnconference/discussions#6] [Ideen / Idées]
Resultate der Diskussion an der GeoUnconference 1 / Résultats de la discussion lors de la GeoUnconference 1

![Marketing for, and visibibility of geodata and geoinformation](https://user-images.githubusercontent.com/4510815/124478744-c9ce1280-dda5-11eb-8118-d7b6ff4dbb5f.jpg)

**Comment by rastrau** [comment]
Ich fände es zu Beginn wertvoll, die **Elemente, die wir der NGDI zuschreiben**, mal zusammenzutragen, um dann weiterzudenken, wie ein gemeinsamer Auftritt aussehen könnte.

Ich glaube, die "Elemente der NGDI" sind nicht allen klar (bzw. werden nicht von allen gleich wahrgenommen), insbesondere nicht potenziellen Nutzerinnen und Nutzern von amtlichen Geodaten ausserhalb des «inneren Kreises» von Geodatenproduzenten, GIS-Fachstellen und -Firmen oder zugewandten Sektoren.

Aus meiner Sicht:
- Geodatenportale des Bundes, der einzelnen Kantone und von Städten / Gemeinden (welche gibt es hier überhaupt, die eine eigene Infrastruktur betreiben?)
- Geodienste des Bundes, der einzelnen Kantone und von Städten / Gemeinden (welche gibt es hier überhaupt, die eine eigene Infrastruktur betreiben?)
- Geodaten des Bundes, der einzelnen Kantone und von Städten / Gemeinden (welche gibt es hier überhaupt, die eine eigene Infrastruktur betreiben?)
- Metaportale:
  - geodienste.ch der Kantone bzw. von KGK
  - geodatenbezogene Teile von opendata.swiss (?)
  - geodatenbezogene Teile von opentransportdata.ch (?)
- Metadatenportal geocat.ch

Etwas weniger klar, aber vielleicht auch Überlegungen wert (aus meiner persönlichen Sicht):
- Standards? Welche?
- Werkzeuge? Welche?
- Online-Repositories von Fachgesetzgebungen wie GeoIG, GeoIV, kGeoIG, kGeoIV, etc.?
- Gremien? Welche?
- Foren? Welche?
- Social Media-Kanäle oder Themen/Hashtags?

Insbesondere für Aussenstehende, aber manchmal auch für Insider schwierige praktische Fragen:
- Welche Daten werden wo vorgehalten?
- Wodurch unterscheiden sich zwei (ähnlich klingende) Datensätze (auf unterschiedlichen Staatsebenen)? Was sind die komparativen Vorteile? (in meinem Verständnis häufig Datenmodellharmonisierung vs. Aktualität) Welcher Datensatz ist für welchen Zweck zu verwenden?

Weitere Anhaltspunkte geben allenfalls die **Einschätzungen zur NGDI**, die wir anlässlich der [GeoUnconference 1](https://github.com/GeoUnconference/geounconference-1) zusammengetragen haben:

![image](https://user-images.githubusercontent.com/4510815/124751728-44726b80-df27-11eb-97c6-0b4ca5a4ce81.png)

**Reply by mizehnder** [reply]
Um ggf. eine Diskussion anzuregen. :)
Bzgl. Elemente der NGDI unterstütze ich die erste Aufzählung der Geoportale, Geodienste, Geodaten, Metaportale und Metadaten. 
Ausser einem Punkt, dass ich geodienste.ch nicht als Metaportal bezeichnen würde. Natürlich stammen die Ursprungsdaten von den Kantonen, doch die schweizweiten Darstellungsdienste und Datendownloads sind bei den einzelnen Kantonen nicht vorhanden.
Ich verstehe die NGDI / das NGDI-Portal als Metaportal, welches auf die einzelnen bestehenden Elemente verweist.

Bzgl. 
> - Online-Repositories von Fachgesetzgebungen wie GeoIG, GeoIV, kGeoIG, kGeoIV, etc.?

Etwas in dieser Art? https://www.geolex.ch/ 


> - Welche Daten werden wo vorgehalten?

Genau das müsste die NGDI vermitteln können. 
> - Wodurch unterscheiden sich zwei (ähnlich klingende) Datensätze (auf unterschiedlichen Staatsebenen)? Was sind die komparativen Vorteile? (in meinem Verständnis häufig Datenmodellharmonisierung vs. Aktualität) 

Aktuell ist es wohl häufig tatsächlich Harmonisierung vs. Aktualität (aufgrund noch fehlender Automatisierung bei den Publikationsprozessen), doch eigentlich sollte es doch Harmonisierung vs. Spezialisierung sein. Um einheitliche Daten über zwei Staatsebenen hinweg zu erhalten, braucht es eine Harmonisierung, meist verbunden mit einem Informationsverlust. Auf kleinerem Raum, also kommunal oder kantonal, sind aber ggf. andere Informationen vorhanden und von Interesse wie auf nationaler Ebene. Um kleinräumige Prozesse untersuchen und abbilden zu können, sind auch verschiedene Datenmodelle notwendig. Ich beobachte aber eher eine allgemeine Harmonisierung über die Staatsebenen hinweg. Ist mal ein harmonisiertes Datenmodell auf nationaler Ebene vorhanden, richten sich die kantonalen und kommunalen Modelle danach aus.

**Reply by rastrau** [reply]
Danke für die Anregungen 🙂 Und Danke für die Präzisierung bezüglich geodienste.ch. Stimmt natürlich, auch wenn es aus Nutzendensicht wohl diesen Charakter z.T. hat.

Genau, an https://geolex.ch habe ich u.a. auch gedacht. 

Die Harmonisierungsdiskussion ist spannend. Mit der Beschleunigung von Produktions- und Nachführungsprozessen sollte es gelingen, den aktuell (gefühlten) Gradienten von harmonisiert bis aktuell zum eigentlich beabsichtigten harmonisiert bis spezialisiert zu verschieben. Den letzteren gibt es natürlich schon, aber weil die harmonisierten Daten zum Teil «langsam» sind, wird bei den spezialisierten Daten häufig die höhere Aktualität als wertvoll wahrgenommen. 

Die NGDI besteht aus wirklich vielen Teilen. Leider, glaube ich, benötigen die meisten Leute inklusive ich lange Zeit, bis sie die Zusammenhänge einigermassen oder gar gut verstehen. Und gerade bei der Suche nach Daten sind die Abwägungen z.T. nicht trivial und man muss bisweilen Daten «in die Hand nehmen» um z.B. auch die Auswirkungen unterschiedlicher Zeitstände abschätzen zu können. Eine NGDI-Nutzung «en passant» ist im Moment in vielen Fällen noch (zu) schwierig, weshalb wir es idealerweise schaffen sollten, die Komponenten der NGDI und ihr Zusammenwirken besser zu erklären.

**Comment by rastrau** [comment]
Die Gruppe an der GeoUnconference 2 hat folgende Lösungsskizze vorgestellt (leider etwas unscharf fotografiert, sorry): 

Idee für eine Marke: **geo.swiss**

![image](https://user-images.githubusercontent.com/4510815/133232125-4d85d81d-af4d-4dc9-adee-b21ff5b8af32.png)

Der "gedruckte" Text wurde in der Gruppe kontrovers diskutiert und während der Präsentation ausgewischt ;-)
In Grün sind diverse Websites / Portale der "Geo-Landschaft" Schweiz aufgeführt: geo.admin.ch, geodienste.ch, geolex.ch

**Comment by rastrau** [comment]
An der GeoUnconference 4 hat eine Gruppe um @kuhnpa, Beat Tschanz, Arnaud van Mesdag und Patrick Ibele weiter am Thema gearbeitet. Ihre Resultate hat die Gruppe [in einem kurzen Bericht zusammengefasst](https://github.com/GeoUnconference/geounconference/blob/main/assets/geounconference-4/topic-8-report.pdf): 

[![](https://raw.githubusercontent.com/GeoUnconference/geounconference/main/assets/geounconference-4/topic-8-report-title-page.png)](https://github.com/GeoUnconference/geounconference/blob/main/assets/geounconference-4/topic-8-report.pdf)

(Der Bericht ist neben PDF auch [als DOCX verfügbar](https://github.com/GeoUnconference/geounconference/raw/main/assets/geounconference-4/topic-8-report.docx))
